openassessment/workflow/migrations/0003_remove_uuid_test.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workflow', '0002_remove_django_extensions'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='assessmentworkflow',
+            name='uuid',
+        ),
+    ]
```

Comment:
Ew

https://docs.djangoproject.com/en/1.9/howto/writing-migrations/#migrations-that-add-unique-fields